egui_plot/src/legend.rs

```diff
@@ -86,6 +99,25 @@ impl Legend {
         self.hidden_items = Some(hidden_items.into_iter().collect());
         self
     }
+
+    /// Specifies if the legend item order should be the inserted order.
+    /// Default: `false`.
+    /// If `true`, the order of the legend items will be the same as the order as they were added.
```

Comment:
```suggestion
    /// If `true`, the order of the legend items will be the same as the order as they were added.
    /// By default it will be sorted alphabetically.
```
I think we should at least mention the default order somewhere